### PR TITLE
Add version upgrade check for mozilla-nss

### PR DESCRIPTION
**Pull request by jouyingbin**
According to 1192086 - FIPS: NSS: Provide module name/identifier and version, mozilla-nss needs to be upgrade to 3.68 for SP4 FIPS certification.

- Related ticket: https://progress.opensuse.org/issues/104989
- Needles: NA
- Verification run: 
  http://openqa.suse.de/tests/8018941#step/firefox_nss/15
  http://openqa.suse.de/tests/8018941#step/firefox_nss/18

### CODE REVIEW

File: tests/fips/mozilla_nss/firefox_nss.pm [hunk 1]
```diff
@@ -68,6 +71,19 @@ sub run {
     my $firefox_version = script_output(q(rpm -q MozillaFirefox | awk -F '-' '{ split($2, a, "."); print a[1]; }'));
     record_info('MozillaFirefox version', "Version of Current MozillaFirefox package: $firefox_version");
 
+    if (get_var('FIPS_ENABLED')) {
+        zypper_call('in mozilla-nss mozilla-nss-certs');
+        my $mozilla_nss_ver = script_output("rpm -q --qf '%{version}\n' mozilla-nss");
```
Comment (rfan1) [inline tests/fips/mozilla_nss/firefox_nss.pm:76]: @jouyingbin  I suggest using 'package_version_cmp', but not must.

Comment (jouyingbin) [reply tests/fips/mozilla_nss/firefox_nss.pm:76]: ah yes, it is a nicer way :) I will modify it, thanks.

File: tests/fips/mozilla_nss/firefox_nss.pm [hunk 2]
```diff
@@ -68,6 +71,19 @@ sub run {
...
+
+        # mozilla-nss attempt to update to 3.68+ in SLE15 SP4 base on FIPS certification 140-3
+        if (!is_sle('<15-sp4') && ($mozilla_nss_ver ge 3.68)) {
+            record_info('mozilla_nss version', "Version of Current openssl package: $mozilla_nss_ver");
```
Comment (rfan1) [inline tests/fips/mozilla_nss/firefox_nss.pm:80]: BTW, **version of Current openssl package** , openssl may be typo? should be mozilla_nss, correct?

Comment (jouyingbin) [reply tests/fips/mozilla_nss/firefox_nss.pm:80]: @rfan1 yes, this is definitely a typo, thanks for finding :)

**Review by lilyeyes — APPROVED** [review]
lgtm

File: tests/fips/mozilla_nss/firefox_nss.pm (hunk 1 above)
Comment (jouyingbin) [reply tests/fips/mozilla_nss/firefox_nss.pm]: change to package_upgrade_check()

File: tests/fips/mozilla_nss/firefox_nss.pm (hunk 2 above)
Comment (jouyingbin) [reply tests/fips/mozilla_nss/firefox_nss.pm]: change to package_upgrade_check()

**Review by rfan1 — APPROVED** [review]
LGTM

**Pull request merged**